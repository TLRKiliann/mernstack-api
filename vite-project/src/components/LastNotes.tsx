import { Component } from 'react'
import notesType from './Model/notesType'
import SubLastComp from './last/SubLastComp'

export default class LastNotes extends Component<notesType> {
  constructor(props) {
    super(props)
  }
  render() {
    return(
      <div>
        1 - Hello Last ! {this.props.lastname}
        <SubLastComp subLastname={this.props.lastname}/>
      </div>
    )
  }
}