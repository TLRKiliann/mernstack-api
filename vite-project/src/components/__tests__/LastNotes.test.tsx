//import React from "react";
import {beforeEach, describe, expect, test, it, vi} from 'vitest';
import {fireEvent, render, screen, waitFor} from '@testing-library/react';
//import "@testing-library/jest-dom/extend-expect"
//import "@testing-library/jest-dom";
import { act } from 'react-dom/test-utils';
import {create} from 'react-test-renderer';
import LastNotes from "../LastNotes.tsx";

test('MatchSnapShot test LastNotes 1', () => {
	const tree = create(<LastNotes />)
	expect(tree.toJSON()).toMatchSnapshot();
})