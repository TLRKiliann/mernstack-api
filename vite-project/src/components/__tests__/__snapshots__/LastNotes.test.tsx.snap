// Vitest Snapshot v1

exports[`MatchSnapShot test LastNotes 1 1`] = `
<div>
  1 - Hello Last ! 
  <div
    style={
      {
        "color": "white",
      }
    }
  >
    2 - SubComponent subLast: 
  </div>
</div>
`;
